client/landing/gutenboarding/index.tsx

```diff
@@ -139,7 +139,6 @@ function waitForSelectedSite(): Promise< Site | undefined > {
  */
 function ensureRandomizedDesignsAreUpToDate() {
 	const designsInStore = select( ONBOARD_STORE ).getRandomizedDesigns();
-	const availableDesigns = getAvailableDesigns();
```

Comment:
@ciampo, could you provide some context for rolling back this change? When we'll be implementing the first A/C of https://github.com/Automattic/wp-calypso/issues/51404 we'll need a way to call `getAvailableDesigns` with some argument so it would return the randomized list of designs. Of course, exposing both the function and the pre-computed list of designs could be an option.

Reply:
We rolled it back because we're exposing `availableDesigns` directly (see line 14), which is basically the same, but with the computation done only once directly in the `@automattic/design-picker` module

Reply:
It's fine to leave it like this. Ultimately, I'm 100% for keeping the scope of this extraction PR somewhat contained. We'll cross that bridge when we come to it.